web/src/user_profile.js

```diff
@@ -349,6 +349,7 @@ export function show_user_profile(user, default_tab_key = "profile-tab") {
         !people.is_my_user_id(user.user_id);
     const args = {
         user_id: user.user_id,
+        is_active: people.is_person_active(user.user_id),
         full_name: user.full_name,
         email: user.delivery_email,
         profile_data,
```

Comment:
Can you put up a follow-up PR sorting these parameters alphabetically? I think that'd make this a lot nicer at this point than the semi-random ordering it has now.

Reply:
Added [a commit](https://github.com/zulip/zulip/pull/26954/commits/bef4a4bae62b618ce4a3d369361861fbb47babe0) for this issue.